recipes/qpdf/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,9 @@
+#include <qpdf/DLL.h>
+#include <iostream>
+
+int
+main(int argc, char* argv[])
+{
+    std::cout << "QPDF_VERSION " << QPDF_VERSION << "\n";
```

Comment:
It's not enough. QPDF_VERSION is a define so we are consuming only a header, but not testing the library linkage. Please, add a function from the library to this code.

Reply:
Ok yeah, also good point. I added now an as minimal as possible write-out of an empty pdf to ensure that there is run-time dependency of linked functionality that can't be optimized away. I hope the new test_package content is suitable now